shared/src/common/enums/http/http-code.enum.ts

```diff
@@ -5,6 +5,7 @@ enum HttpCode {
   UNAUTHORIZED = 401,
   NOT_FOUND = 404,
   INTERNAL_SERVER_ERROR = 500,
+  DELETED = 204,
```

Comment:
```suggestion
  NO_CONTENT = 204,
```
let's give them common names